### DISCUSSION TITLE
Ide for develop

### DISCUSSION THREAD

**Discussion by vitalinvent** [gabonator/LA104#101] [General]
Which ide for develop VIM, IAR, Code?
- code completion
- highlighting code
- find declarations
does anybody have any suggestions?

**Comment by gabonator** [comment]
On OSX XCode is a good choice, some projects have `build_mac.sh` script which uses cmake to produce xcode compatible project. Just change the `-G` switch to your preference and you are ready to go. On Windows I would recommend using Microsoft Visual Studio Code:

```sh
mkdir build
cd build
cmake .. -DDESKTOP=1 -DCMAKE_INSTALL_PREFIX=../_install -GXcode
```

<img width="1073" alt="image" src="https://user-images.githubusercontent.com/1020041/209342176-abd3d92e-e8f1-410e-b0c2-54c2ae799bd6.png">

**Reply by vitalinvent** [reply]
thank you, i'll try in hackintosh emul run xcode :)
which version xcode min, can i use some old  versions?

**Reply by gabonator** [reply]
Don't even think about it. There is visual studio offering same features (or even more):
https://github.com/gabonator/LA104/discussions/52#discussioncomment-4489201
